### PR TITLE
[Security][Validator] Missing translations for Luxembourgish

### LINKED ISSUE #51899: Missing translations for Luxembourgish (lb)
Hello,

There are some missing Luxembourgish (lb) translations and we are looking for a **native** speaker to help us out. 

Here is a [short example](https://symfony-translations.nyholm.tech/#pr) of what you need to do. There are 4 rules: 

1. You must be a Luxembourgish (lb) native speaker
2. You must look at the existing translations and follow the same "style" or "tone"
3. You must make your PR to branch 5.4
4. You must use the correct indentation (number of spaces)

These are the files that should be updated: 
- [src/Symfony/Component/Security/Core/Resources/translations/security.lb.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Security/Core/Resources/translations/security.lb.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf)

> [!NOTE]
> If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by crownbackend**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Issues        | Fix #51899 
| License       | MIT

Added missing translations validator (translation units 100-109).
Added missing translations security (translation units 10-20).

### CODE REVIEW

File: src/Symfony/Component/Security/Core/Resources/translations/security.lb.xlf [hunk 1]
```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Ongëltegen oder ofgelafene Login-Link.</target>
             </trans-unit>
+            <trans-unit id="19">
+                <source>Too many failed login attempts, please try again in %minutes% minute.</source>
+                <target>Zu vill fehlgeschloen Aloggin Versich, w.e.g. probéiert nach am %minutes% Minutt.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Core/Resources/translations/security.lb.xlf]: WDYT of this?
```suggestion
                <target>Zu vill fehlgeschloen Loginversich, w. e. g. probéiert nach am %minutes% Minutt.</target>
```

File: src/Symfony/Component/Security/Core/Resources/translations/security.lb.xlf [hunk 2]
```diff
@@ -70,6 +70,14 @@
...
+            </trans-unit>
+            <trans-unit id="20">
+                <source>Too many failed login attempts, please try again in %minutes% minutes.</source>
+                <target>Zu vill fehlgeschloen Aloggin Versich, w.e.g. probéiert nach am %minutes% Minutten.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Security/Core/Resources/translations/security.lb.xlf]:
```suggestion
                <target>Zu vill fehlgeschloen Loginversich, w. e. g. probéiert nach an %minutes% Minutten.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 3]
```diff
@@ -386,6 +386,46 @@
                 <source>This value is not a valid International Securities Identification Number (ISIN).</source>
                 <target>Dëse Wäert ass keng gëlteg International Wäertpabeiererkennnummer (ISIN).</target>
             </trans-unit>
+            <trans-unit id="100">
+                <source>This value should be a valid expression.</source>
+                <target>Dëse Wäert soll eng gëlteg Expression sinn.</target>
+            </trans-unit>
+            <trans-unit id="101">
+                <source>This value is not a valid CSS color.</source>
+                <target>Dëse Wäert ass keeng gëlteg CSS Faarf.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]:
```suggestion
                <target>Dëse Wäert ass keng gëlteg CSS Faarf.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 4]
```diff
@@ -386,6 +386,46 @@
...
+            </trans-unit>
+            <trans-unit id="102">
+                <source>This value is not a valid CIDR notation.</source>
+                <target>Dëse Wäert ass keeng gëlteg CIDR Notatioun.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]:
```suggestion
                <target>Dëse Wäert ass keng gëlteg CIDR Notatioun.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 5]
```diff
@@ -386,6 +386,46 @@
...
+                <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
+                <target>De Wäert vum Netmask soll tëscht {{ min }} a {{ max }} sinn.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>De Dateinimm ass ze laang. Et sollt {{ filename_max_length }} Zeechen oder manner hunn.|De Dateinimm ass ze laang. Et sollt {{ filename_max_length }} Zeechen oder manner hunn.</target>
+            </trans-unit>
+            <trans-unit id="105">
+                <source>The password strength is too low. Please use a stronger password.</source>
+                <target>D'Lëschtungsstaarf ass ze nieft. Benotzt w.e.g. e stäerker Passwuert.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]:
```suggestion
                <target>D'Staarf vum Passwuert ass ze schwaach. Benotzt w. e. g. e stäerker Passwuert.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 6]
```diff
@@ -386,6 +386,46 @@
...
+            </trans-unit>
+            <trans-unit id="106">
+                <source>This value contains characters that are not allowed by the current restriction-level.</source>
+                <target>Dëse Wäert enthält Zeechen, déi net erlaabt sinn no der aktueller Bescchränkungsstuf.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]:
```suggestion
                <target>Dëse Wäert enthält Zeechen, déi net erlaabt sinn no der aktueller Beschränkungsstuf.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 7]
```diff
@@ -386,6 +386,46 @@
...
+            </trans-unit>
+            <trans-unit id="107">
+                <source>Using invisible characters is not allowed.</source>
+                <target>D'Benotzen vu onsichten Zeechen ass net erlaabt.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]:
```suggestion
                <target>D'Benotzen vu onsiichtbaren Zeechen ass net erlaabt.</target>
```

**Review by nicolas-grekas — COMMENTED** [review]
Thanks for the PR, here are some suggestions. Would they make sense to you?

**Comment by crownbackend** [comment]
hey, i accepte you change, sorry i m realy bad orthography

**Comment by nicolas-grekas** [comment]
Thank you @crownbackend.

**Pull request merged**